packages/now-node/src/index.ts

```diff
@@ -361,10 +361,7 @@ export async function build({
     meta,
   });
 
-  debug('Running user script...');
-  const runScriptTime = Date.now();
-  await runPackageJsonScript(entrypointFsDirname, 'now-build', spawnOpts);
```

Comment:
@leo @styfle Previously, only `now-build` was considered in `@vercel/node` (`build` is never executed). It's not clear to me if this is intentional or not, but this PR updates to invoke `vercel-build`/`now-build`/`build`. Please advise on if this is the correct behavior.

Reply:
I think we will want to avoid this. Many users have a `build` script now that we introduced zero config and that script is not intended for serverless functions, so we might slow down existing deployments by changing this

Reply:
Ya, makes sense. I reverted back to the current behavior.